### PR TITLE
feat(firefox): apply emulation to all pages in the browser context

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: test/popup.spec.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      await context.close();
+      expect(request.headers['user-agent']).toBe('hey');
+    });
+    it.skip(CHROMIUM)('should inherit touch support from browser context', async function({newContext, server}) {
+      const context = await newContext({
+        viewport: { width: 400, height: 500, isMobile: true }
+      });
+      const page = await context.newPage();
+      await page.goto(server.EMPTY_PAGE);
+      await new Promise(f => setTimeout(f, 500));
```
Comment (aslushnikov) [inline test/popup.spec.js]: oh!

File: test/popup.spec.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      const popupPromise = page.waitForEvent('popup');
```
Comment (aslushnikov) [inline test/popup.spec.js]: this is not needed?

File: test/popup.spec.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      await context.close();
+      expect(hasTouch).toBe(true);
+    });
+    it.skip(CHROMIUM)('should inherit viewport size from browser context', async function({newContext, server}) {
+      const context = await newContext({
+        viewport: { width: 400, height: 500, isMobile: true }
+      });
+      const page = await context.newPage();
+      await page.goto(server.EMPTY_PAGE);
+      await new Promise(f => setTimeout(f, 500));
```
Comment (aslushnikov) [inline test/popup.spec.js]: ditto

File: test/popup.spec.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      const popupPromise = page.waitForEvent('popup');
```
Comment (aslushnikov) [inline test/popup.spec.js]: ditto

**Review by aslushnikov — APPROVED** [review]

File: test/popup.spec.js (hunk 1 above)
Comment (dgozman) [reply test/popup.spec.js]: Fixed

File: test/popup.spec.js (hunk 2 above)
Comment (dgozman) [reply test/popup.spec.js]: Yep

**Pull request merged**